show/main.py

```diff
@@ -1917,6 +1917,210 @@ def mmu():
     click.echo(proc.stdout.read())
 
 
+#
+# 'fg nhg active-hops' command ("show fg-nhg-active-hops")
+#
+@cli.command('fg-nhg-active-hops')
+@click.argument('nhg', required=True)
+def fg_nhg_active_hops(nhg):
+    state_db = SonicV2Connector(host='127.0.0.1')
```

Comment:
please use pass_db decorator.

Reply:
For some reason, when locally testing using pass_db decorator I wasn't able to fetch my config_db table using "db.cfgdb.get_table()".

Only when using SonicV2Connector was I able to fetch my config_db table. Any suggestions?